hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/delegation_tokens.md

```diff
@@ -353,10 +352,10 @@ it is authenticated with; the role token binding will fail.
 
 When the AWS credentials supplied to the Session Delegation Token binding
 through `fs.s3a.aws.credentials.provider` are themselves a set of
```

Comment:
Something is off here, maybe "they themselves are part of session credentials."

Reply:
this seems fine to me as is. Maybe removing the themselves entirely would help, so it becomes "through `fs.s3a.aws.credentials.provider` are a set of", but personally feel it's clearer as is.